src/Analysis/GWASV2/Components/Covariates/ContinuousCovariates.jsx

```diff
@@ -13,35 +21,58 @@ const ContinuousCovariates = ({ dispatch, handleClose, submitButtonLabel }) => {
 
   return (
     <React.Fragment>
-      <Covariates selected={selected} handleSelect={setSelected} />
-      <button
-        type='button'
-        style={{ marginLeft: 5 }}
-        onClick={() => {
-          dispatch(formatSelected(selected));
-          handleClose();
-        }}
-      >
-        {submitButtonLabel}
-      </button>
-      <button
-        type='button'
-        onClick={() => {
-          handleClose();
-        }}
-      >
-        Cancel
-      </button>
+      <div className='GWASUI-flexRow' style={{ width: '950px' }} data-tour='name'>
+        <div>
+          <div>
+            <Covariates selected={selected} handleSelect={setSelected} />
+            <button
+              type='button'
+              style={{ marginLeft: 5 }}
+              onClick={() => {
+                dispatch(formatSelected(selected));
+                handleClose();
+              }}
+            >
+              {submitButtonLabel}
+            </button>
+            <button
+              type='button'
+              onClick={() => {
+                handleClose();
+              }}
+            >
+            Cancel
+            </button>
+          </div>
+        </div>
+        <div style={{ paddingLeft: '30px' }}>
```

Comment:
I think this is fine for now, but I plan on replacing the inline styles with CSS class names.

Reply:
I'm counting on a final styling by you! Thanks 👍